### ISSUE TITLE
Any support to disable `Rails/WhereRange` ?

### ISSUE THREAD

**Issue by stevegeek** [standardrb/standard-rails#56]
Since `rubocop-rails` 2.25 a new cop [`Rails/WhereRange` was introduced](https://github.com/rubocop/rubocop-rails/pull/1272). I see it was enabled in [`standard` here](https://github.com/standardrb/standard-rails/pull/40)

Just wanted to tentatively put out why I think it maybe should have been disabled in `standard-rails` instead.

While I agree to discourage using SQL string fragments, I'm not sure that beginless ranges make readable alternatives, eg `User.where('age < ?', 18)` becomes `User.where(age: ...18)` 

I imagine `<` is more widely understandable as "less than" compared to, "range up to but not including 18" (which one must remember that `...` mean exclusive end).

Also when you consider `<=` vs `<` I find 2 dots vs 3 dots range literal syntax here hides the intent of the code more than its original form, eg:

```
User.where('age <= ?', 18).where('height < ?', 200)
User.where(age: ..18).where(height: ...200)
```

Finally, Ranges always include their beginning, so there is no Range `where` clause form for greater than, ie `User.where('age > ?', 18)` , so either way those versions will remain in your codebase, eg

`User.where('age <= ?', 18).where('height > ?', 200)` becomes `User.where(age: ..18).where('height > ?', 200)`

Thanks for the consideration and for all the work on `standard`!!

**Comment by searls** [comment]
I agree, I actually missed this in the original PR -- my bad. Would support disabling :+1:

**Comment by stevegeek** [comment]
@searls ok cool. I tried to get a bit of feedback from other devs on this, and no one seems particularly pro it, so I will open a PR to disable it
